### PR TITLE
Remove dependency and add grunt-cli

### PR DESCRIPTION
**Pull request by geek**
- Removed dependency on core-util-is
- Added grunt-cli as a dependency so that contributors aren't forced into installing globally or separately

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,13 +35,12 @@
     "errorhandler": "^1.1.1",
     "express": "^4.3.8",
     "express-session": "^1.7.5",
-    "grunt": "~0.4.1",
+    "grunt": "~0.4.5",
+    "grunt-cli": "~1.2.0",
```
Comment (indutny) [inline package.json]: Any point to not use `^` here?

**Comment by indutny** [comment]
What is a reason for the removal of `core-util-is`? Why was it added in first place?

File: package.json (hunk 1 above)
Comment (geek) [inline package.json]: None that I see, was going off of the other version usage for grunt

**Comment by geek** [comment]
@indutny Removing because it seems like overkill for 3 type checks.  The only issue I see with this PR is that if need to support old versions of node (pre 0.8.0) that don't have `Array.isArray`

**Comment by indutny** [comment]
@geek I doubt there is any reason to support pre 0.8.0 nowadays.

Please change `package.json`, though ;)

👍

**Comment by grawk** [comment]
Seems fine. I'll give anyone a chance to chime in as far as installing a local copy of grunt. This is a change that we'd want to make all over if we're going to start making it.

**Comment by jasisk** [comment]
Cool on all things.

`core-util-is` usage was more of a common pattern amongst our modules than it is a matter of practical usage. This change is a bit _change for changes sake_ but still practical.

`grunt-cli` has been added in some of our other repos—`lusca` has just been stuck in the _prefer global_ mindset from nearly four years ago. `lusca` is amongst the top offenders as far as avoiding spring-cleaning has gone. 😀

**Comment by linkRace** [comment]
Seems like this has support to merge?  Will get one more from my team.

**Comment by subeeshcbabu** [comment]
👍

**Pull request merged**